test-apps/display-test-app/src/frontend/ClassificationsPanel.ts

```diff
@@ -23,6 +23,16 @@ function clearElement(element: HTMLElement): void {
 
 const NO_MODEL_ID = "-1";
 
+enum RealityDataType {
```

Comment:
I think I've seen this enum before...did it move somewhere that display-test-app (and other apps) cannot access it? Is that why it's duplicated here?

Reply:
You probably see this in the old client. I'm not sure where I should put this. What do you think if we add an enum called SupportedRealityDataType in ContextRealityModel?

Reply:
Seems like the enum ought to be provided by the same package that provides ITwinRealityData - reality-data-client.

Reply:
Removed, will be added at the correct place